nf_core/pipeline-template/nextflow.config

```
@@ -81,7 +81,10 @@ try {
 {% endif %}
 
 profiles {
-    debug { process.beforeScript = 'echo $HOSTNAME' }
+    debug {
+        dumpHashes             = true
+        process.beforeScript   = 'echo $HOSTNAME'
```

```suggestion
        process.beforeScript   = 'echo $HOSTNAME'
        cleanup = false
```
Suggested by @jfy133